unstoppable-ios-app/domains-manager-ios/NetworkEnvironment/NetworkConfig.swift

```diff
@@ -58,6 +58,14 @@ struct NetworkConfig {
         }
     }
     
+    static var baseProfileAPIUrl: String {
```

Comment:
We already have a var called `baseProfileHost`

Need to differentiate these more I guess